src/components/panels/base/coordinator.tsx

```diff
@@ -165,6 +165,7 @@ export function usePanelsCoordinator<L extends string>(props: PanelCoordinatorPr
             }
         } catch (e) {
             // Do nothing.
+            console.error(e)
```

Comment:
Sending it to the console in case someone wants to look, which also silences an unused var lint error.